### DISCUSSION TITLE
Zemismart ZM25TQ Not showing correct Power Info

### DISCUSSION THREAD

**Discussion by dsduarte** [Koenkk/zigbee2mqtt#14689] [Devices]
Hello!!
I have recently added 2 new tubular motor on my network and one of them is not showing the power icon properly (as showing on image).
I have tryied to remove/add several times and got the same result.
Any clues on what is going on?

![z2m-motor](https://user-images.githubusercontent.com/19639490/198604972-8924c25d-6b33-4b7d-a45e-d3250f1d32a4.png)

**Comment by dsduarte** [comment]
I was waiting a few days to see if that situation could change but nothing so far.... one of them are still showing with a question mark on power info.

**Comment by antti365** [comment]
Hello, did you find a solution? I have the same problem, 2 out of 4 motors show question marks, on older versions of z2m all four motors showed up fine.
To @Koenkk ver. 1.34.0 commit: 56589dc

**Comment by dsduarte** [comment]
hi... no... still the same here.... Scince that post I added a 3rd device and its reporting ok...

**Comment by antti365** [comment]
@Koenkk Hi, is there any way to solve this, it looks like the problem is over a year old...

**Comment by Koenkk** [comment]
Try to force remove the device and re-pair it.

**Reply by antti365** [reply]
I tried it several times, it doesn’t help, so far 2 devices are fine, two have a question mark

**Reply by Koenkk** [reply]
Should it always show AC powered?

**Reply by antti365** [reply]
Well, yes, in the old versions it was like that...
![Снимок экрана 2024-01-05 в 14 32 31](https://github.com/Koenkk/zigbee2mqtt/assets/6865425/c26040ec-e234-4b3f-8d75-0138a55d7338)

**Reply by Koenkk** [reply]
pushed a fix.

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Reply by antti365** [reply]
Thank you very much!
